.github/workflows/ci-mingw.yml

```diff
@@ -0,0 +1,89 @@
+# This is a copy of ci-unix-shared-local.yml for building shared libraries
+# with an additional build configuration (using installed deps and dav1d).
+
+name: CI MINGW
```

Comment:
"MinGW" 😉

Reply:
Right, but I thought (just like on Ubuntu), that it was better to compile gtest from source to make sure the same compile flags are passed?

Reply:
Done, let's be right :)

Reply:
I guess it depends on whether libavif changes the compile options significantly from the ones cmake sets...